data/core/units/orcs/Adept.cfg

```diff
@@ -13,17 +16,15 @@
     experience=50
     level=1
     alignment=chaotic
-    advances_to=null
-    {AMLA_DEFAULT}
-    cost=21
+    advances_to=Orcish Shaman
+    cost=20
     usage=mixed fighter
-    description= _ "Orcish Shamans are the guardians of orcish magic. Respected among orcish tribes, they form the Orcish Council, which makes important decisions for the whole Orcish community and arbitrates the many conflicts that arise between tribes of this argumentative race. Although physically weak (for orcs), Orcish Shamans are good spell casters and can curse their enemies, draining their life. Novice Shamans are still young and vigorous, but their spell-casting ability still needs improvement."
+    description= _ "Orcish Shamans are the guardians of orcish magic. Respected among orcish tribes, they form the Orcish Council, which makes important decisions for the whole Orcish community and arbitrates the many conflicts that arise between tribes of this argumentative race. Although physically weak (for orcs), Orcish Shamans are good spell casters and can curse their enemies, draining their life. Adepts are the novice shamans, still young and vigorous, but their spell-casting ability still needs improvement."
```

Comment:
Many of the translatable strings should have `# po:` hints that this unit-line is female-only